include/swift/Serialization/ModuleFile.h

```diff
@@ -336,7 +336,7 @@ class ModuleFile : public LazyMemberLoader {
     // Explicitly pad out to the next word boundary.
     unsigned : 0;
   } Bits = {};
-  static_assert(sizeof(Bits) <= 8, "The bit set should be small");
+  static_assert(sizeof(Bits_Struct_Name) <= 8, "The bit set should be small");
```

Comment:
It doesn't seem to support `sizeof(value)` well in many places. It's always happy with `sizeof(TypeName)` though...
I've renamed the struct as you suggested.